src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/EtwInput.cs

```diff
@@ -0,0 +1,163 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Diagnostics.EventFlow.Configuration;
+using Microsoft.Diagnostics.EventFlow.Utilities.Etw;
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Session;
+using Microsoft.Extensions.Configuration;
+using Validation;
+
+namespace Microsoft.Diagnostics.EventFlow.Inputs
+{
+    public class EtwInput : IObservable<EventData>, IDisposable, IRequireActivation
+    {
+        private IHealthReporter healthReporter;
+        private EventFlowSubject<EventData> subject;
+        private ITraceEventSession session;
+        private bool isDisposed;
+
+        public EtwInput(IConfiguration configuration, IHealthReporter healthReporter)
+        {
+            Requires.NotNull(configuration, nameof(configuration));
+            Requires.NotNull(healthReporter, nameof(healthReporter));
+
+            IConfiguration providersConfiguration = configuration.GetSection("providers");
```

Comment:
There are 2 places of the same string of 'providers' (in line 31 and line 34). I don't feel conformable seeing strings duplicated themselves in code. :-) Can we refactor that these strings are provided by EtwProviderConfiguration class as constants?